oviewer/content.go

```diff
@@ -241,11 +241,11 @@ func (st *parseState) backspaceHandle() {
 	st.bsFlag = true
 	st.bsContent = st.lc.last()
 
-	if st.bsContent.width == 1 {
-		st.lc = st.lc[:len(st.lc)-1]
+	if st.bsContent.width == 2 && len(st.lc) > 1 {
+		st.lc = st.lc[:len(st.lc)-2]
 		return
 	}
-	st.lc = st.lc[:len(st.lc)-2]
+	st.lc = st.lc[:len(st.lc)-1]
```

Comment:
So the code is about doing this?

```suggestion
	st.lc = st.lc
```

So doing nothing?

```suggestion
```

Reply:
No, the code is not about doing this. The code actually handles backspacing differently depending on the width of the character and the length of the lc slice. 
If the character width is 2 and the length of lc is greater than 1, it removes two elements from lc. Otherwise, it removes one element.

Reply:
Indeed https://go.dev/play/p/e9Tnc2sEYjV